### PR TITLE
Log In

### PR DESCRIPTION
**Pull request by mohdzakwan99**
ready

### CODE REVIEW

**Comment by mohdzakwan99** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,15 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
+  get 'sessions/create'
+  get 'sessions/destroy'
```
Comment (ghost) [inline config/routes.rb]: delete line form 2 to 3 please

**Review by ghost — COMMENTED** [review]
cmt

**Comment by mohdzakwan99** [comment]
ready

File: app/models/user.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
   validates :name, presence: true
   validates :email, uniqueness: true
-  before_save :downcase_email
```
Comment (huongnt-2545) [inline app/models/user.rb]: don't change, keep this

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -1,13 +1,28 @@
-<header class="navbar navbar-fixed-top navbar-inverse">
+  <header class="navbar navbar-fixed-top navbar-inverse">
```
Comment (huongnt-2545) [inline app/views/layouts/_header.html.erb]: - don't change, keep this
- update the indentation from line 2 to the end

**Comment by mohdzakwan99** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull

**Pull request merged**